{"name": "test_login_fail_form_error[data2]", "status": "passed", "attachments": [{"name": "log", "source": "f821f406-488c-4b28-800c-c9898a05cea9-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'user': '', 'password': 'python', 'check': '请输入手机号'}"}], "start": 1567138258062, "stop": 1567138258755, "uuid": "910f08e4-4abb-4e5b-821e-47e7cc9ac3e0", "historyId": "a642135d870fb89ec526453129704811", "testCaseId": "d9574c1722641d515e9f2f722dc354c5", "fullName": "testcases.test_login6.Test_Login#test_login_fail_form_error", "labels": [{"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_login6"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "liliangdeMacBook-Pro.local"}, {"name": "thread", "value": "68435-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_login6"}]}